vault_sdk/src/instruction.rs

```diff
@@ -200,8 +200,14 @@ pub enum VaultInstruction {
         reward_fee_bps: Option<u16>,
     },
 
-    /// Withdraws any non-backing tokens from the vault
-    AdminWithdraw {
+    /// Delegate the token account to a third party
+    #[account(0, signer, name = "admin")]
```

Comment:
ixs here don't match delegate_token_account. can you throw in the config too?

Reply:
Thanks!. I changed to pass the `config` as well.